drivers/mrf24j40/include/mrf24j40_internal.h

```diff
@@ -114,6 +114,27 @@ void mrf24j40_update_tasks(mrf24j40_t *dev);
  */
 void mrf24j40_hardware_reset(mrf24j40_t *dev);
 
+/**
+ * @brief   Enable External Power Amplifier & Low Noise Amplifier
+ *
+ * @param[in] dev       device to enable the PA & LNA on
+ */
+void mrf24j40_enable_pa_lna(mrf24j40_t *dev);
+
+/**
+ * @brief   Disable External Power Amplifier & Low Noise Amplifier
+ *
+ * @param[in] dev       device to disable the PA & LNA on
+ */
+void mrf24j40_disable_pa_lna(mrf24j40_t *dev);
+
+/**
+ * @brief   Enable only the External Low Noise Amplifier
+ *
+ * @param[in] dev       device enable the LNA on
+ */
+void mrf24j40_enable_lna(mrf24j40_t *dev);
```

Comment:
`pa_lna` needs en-/disable but `lna` only needs enable?

Reply:
The idea is that power amplifier and low noise amplifier are always turned on / off together, except for CSMA where only the low noise amplifier is enabled. See [Section 1.3.2 ENERGY DETECTION](http://ww1.microchip.com/downloads/en/devicedoc/75002a.pdf).